packages/webhooks/src/features/event/service.ts

```diff
@@ -132,6 +148,26 @@ export async function transformDeathBundle(
   }
 }
 
+const getPermissionsBundle = async (
+  bundle: fhir.Bundle,
+  permissions: string[] = [],
+  composition: fhir.Composition,
+  authHeader: IAuthHeader
+) => {
+  const resources = await Promise.all(
+    permissions.map((sectionCode) =>
+      getResourceBySection(composition, sectionCode, authHeader)
+    )
+  )
+  resources.map((resource: fhir.BundleEntry) => {
+    if (resource) {
+      bundle.entry!.push({ resource })
+    }
+  })
```

Comment:
nitpick: `.map` is usually used when the return value of `.map` is handled in some way.

In this case `.forEach` would be more fitting when mutating external arrays/objects and the return value of the loop itself is discarded.